Documentation/CRDs/Object-Storage/ceph-object-realm-crd.md

```diff
@@ -29,4 +29,7 @@ spec:
 ### Spec
 
 * `pull`: This optional section is for the pulling the realm for another ceph cluster.
-    * `endpoint`: The endpoint in the realm from another ceph cluster you want to pull from. This endpoint must be in the master zone of the master zone group of the realm.
+    * `endpoint`: The endpoint in the realm from another ceph cluster you want to pull from. This endpoint must be in the master zone of the    master zone group of the realm.
+* `defaultRealm`: (optional) When set to true, Rook will mark the CephObjectStore's realm as the default realm in the Ceph cluster.
```

Comment:
We don't usually specify "optional" in the docs. It's implied by the statement about the default value.
```suggestion
* `defaultRealm`: When set to true, Rook will mark the CephObjectStore's realm as the default realm in the Ceph cluster.
```

Reply:
done